### PR TITLE
Ticket preview

### PR DESCRIPTION
**Pull request by agudenko2006**
Merged TicketPreview and TicketPreviewList into a single component.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import React, { useState } from 'react';
```
Comment (agudenko2006) [inline src/App.tsx:1]: Copied from the master branch

**Comment by agudenko2006** [comment]
App.tsx is copied from the master branch, not sure why git complains...

File: src/App.tsx (hunk 1 above)
Comment (agudenko2006) [inline src/App.tsx]: This file is taken from the master branch

Comment (agudenko2006) [reply src/App.tsx]: Can I remove this file from the PR altogether?

**Review by arswarog — APPROVED** [review]

**Pull request merged**